composer.json

```diff
@@ -52,5 +52,10 @@
                 "Courier": "Courier\\Laravel\\Facades\\CourierFacade"
             }
         }
+    },
```

Comment:
do we need this change? I'd suggest undoing changes to composer.json and composer.lock